Python/Tests/Core/WorkspaceInterpreterFactoryTests.cs

```diff
@@ -206,19 +206,20 @@ public class WorkspaceInterpreterFactoryTests {
             Directory.CreateDirectory(workspacePath);
             File.WriteAllText(Path.Combine(workspacePath, "app.py"), string.Empty);
 
-            CreatePythonVirtualEnv(python.InterpreterPath, workspacePath, envName);
+            virtualEnvModule(python.InterpreterPath, workspacePath, envName);
 
             return new WorkspaceTestHelper.MockWorkspaceContext(new WorkspaceTestHelper.MockWorkspace(workspacePath));
         }
 
-        private static void CreatePythonVirtualEnv(string pythonInterpreterPath, string workspacePath, string envName) {
+        internal static void virtualEnvModule(string pythonInterpreterPath, string workspacePath, string envName, string environmentCreationParameter = "venv") {
```

Comment:
`CreatePythonVirtualEnv` was fine for the function name. It's the parameter name that should be `virtualEnvModule` instead of `environmentCreationParameter`

Reply:
Corrected.